src/components/Layout/Header/DesktopNav.tsx

```diff
@@ -15,7 +15,7 @@ interface Props {
 
 const DesktopNav: FunctionComponent<Props> = ({ navLinks, hideGetStartedButton }) => (
     <>
-        <Nav className="left-nav me-auto ml-md-2">
+        <Nav className="left-nav me-auto ml-xl-2">
```

Comment:
I'm really glad you caught this spacing issue on the nav bar. When I tested locally, I did see that the break persists between 1099px-1109px width (below). I am testing on a Macbook.

This is not in scope for the fix in question, but this does make me think about the two CTAs on the nav bar. I have seen chatter in Slack discussing whether "Request a Demo" and "Get Started" are redundant. Today (6/30) we are pushing an update that will redirect the Get Started CTA to `/get-started/self-hosted`, eliminating the decision tree the CTA provides. Perhaps this is a question for our product folks as they consider the second stage of their plan to conceptualize self hosted vs. cloud offerings.

<img width="1092" alt="Screen Shot 2022-06-30 at 9 28 15 AM" src="https://user-images.githubusercontent.com/60713139/176707256-9029f0fd-0684-42f0-8ab1-bffce48cecd0.png">

Reply:
Hm I can't reproduce that. At 1099px the mobile nav kicks in. @bretthayes are you noticing this bug persist?

I do agree the CTAs are a bit redundant!

https://user-images.githubusercontent.com/59381432/176945526-1a029eb7-2c0e-47bf-83e9-572e7b0f3de3.mov

Reply:
I'm not seeing the wrapping bug persist, but I do notice the height of the nav changes slightly which causes a bit of a jump. Since this is a logo update PR, I don't want to introduce scope creep here. So looks good to me for the time being. I can clean up some of the styles/classes in my sticky nav PR since that's related to the nav.